src/mercury-agent/mercury/agent/procedures/hp_firmware.py

```diff
@@ -0,0 +1,259 @@
+# Copyright 2015 Jared Rodriguez (jared.rodriguez@rackspace.com)
```

Comment:
As much as I would love to take credit for everything, the author should probably change to Hussam, Mercury Developers, or Rackspace :)